docs/sematic-vs.md

```diff
@@ -56,35 +56,97 @@ But bear with us, we'll catch up fast!
 
 * **Dynamic graph** – Sematic lets up loop over configurations, run different branches of your pipelines depending on the outcome of upstream steps, and even do hyperparameter tuning.
 
-### Kubeflow Pipelines
+* **Dead simple I/O between steps** – Sematic makes I/O between steps in your pipelines as simple as passing an output of one
+python function as the input of another. Airflow provides APIs like
+[XComs](https://airflow.apache.org/docs/apache-airflow/stable/concepts/xcoms.html) which can pass data between tasks, but involves
+some boilerplate around explicitly pushing/pulling data around, and coupling producers and consumers via named data keys.
 
-* **Barrier to entry** – Using KFP requires quite a bit of knowledge about
+### ... Kubeflow Pipelines
+
+* **Low Barrier to entry** – Using KFP requires quite a bit of knowledge about
   Kubernetes, Docker images, Argo, etc. These things take a lot of time to
   master. Sematic tries to provide working solutions out-of-the-box so that you
   can focus on your expertise instead of infrastructure.
 
-* **Dynamic graph** – In KFP, DAGs are fixed ahead of time. Sematic lets up loop over configurations, run different branches of your pipelines depending on the outcome of upstream steps, and even do hyperparameter tuning.
+* **Dynamic graph** – In KFP, DAGs are fixed ahead of time. While it does provide
+[some mechanisms](https://kubeflow-pipelines.readthedocs.io/en/stable/source/kfp.dsl.html#kfp.dsl.Condition)
+to modify pipeline behavior mid-execution, you are limited by the provided control flow
+statements. Sematic lets you use standard python control flow to loop over configurations,
+run different branches of your pipelines depending on the outcome of upstream steps,
+and even do hyperparameter tuning.
 
-* **Lineage tracking** – KFP does not keep a rigorous track of all assets and artifacts flowing between your steps. You need to implement that yourself on top of it. In Sematic, lineage tracking is a first-class citizen. All inputs and outputs of all Sematic Functions are tracked and versioned.
+* **Lineage tracking** – KFP has some basic tracking of artifacts flowing between your steps, but is limited in the
+type of data that can be tracked in this way. If you want to have tracking for high-level, typed objects like "models"
+instead of just paths to blobs, you need to implement that yourself on top of it. In Sematic, lineage tracking of richly
+typed artifacts is a first-class citizen. All inputs and outputs of all Sematic Functions are tracked and versioned.
 
 * **Semantic UI** – The KFP UI offers barebones tools to monitor and inspect runs, but no rich visualization or advanced features.
 
-### MLFlow
-
-### Dagster
+### ... MLFlow Pipelines
+
+* **Flexibility** - To use
+[MLFlow pipelines](https://www.mlflow.org/docs/latest/pipelines.html#pipeline-templates),
```

Comment:
Same here, no link. But `MLFlow Pipeline`, capital P, since it is a product name.

Reply:
done